package.json

```diff
@@ -13,10 +13,11 @@
   "scripts": {
     "test": "npm run eslint && npm run coverage && npm run integration-test",
     "eslint": "node ./source/markotester source --no-mocha",
-    "unit-test": "./node_modules/.bin/_mocha $(find source -name '*.spec.js')",
+    "eslint-fix": "eslint ./source --fix",
```

Comment:
never have fix flags in your (or mine :-) command list. It does more potential harm than good. And you still have to type-in "-fix" (in the command name itself), saving one "-" won't do much. If you want to fix stuff, explicitly pass `--fix`.

Reply:
It is useful when you want to fix everything. You can still see what change in the `git diff` and verify that it is correct. I never had a problem with it.